runtime/src/main/java/com/dylibso/chicory/runtime/InterpreterMachine.java

```diff
@@ -1630,11 +1620,7 @@ private static void F32_NEG(MStack stack) {
         stack.push(Value.floatToLong(-tos));
     }
 
-    private static void MEMORY_FILL(MStack stack, Instance instance, Operands operands) {
-        var memidx = (int) operands.get(0);
-        if (memidx != 0) {
-            throw new WASMRuntimeException("We don't support multiple memories just yet");
-        }
```

Comment:
is this handled somewhere else now? e.g. validation?

Reply:
yup: https://github.com/dylibso/chicory/blob/87ecca5f6d40fc15668df16885a5ddbf70b85497/wasm/src/main/java/com/dylibso/chicory/wasm/Validator.java#L583-L593